### PR TITLE
KEYCLOAK-10927 Docs for Implement LDAPv3 Password Modify Extended Operation

### PR DESCRIPTION
**Pull request by cachescrubber**

### CODE REVIEW

**Comment by cachescrubber** [comment]
This PR belongs to https://github.com/keycloak/keycloak/pull/6962.

**Review by mposolda — DISMISSED** [review]

File: server_admin/topics/user-federation/ldap.adoc [hunk 1]
```diff
@@ -167,5 +167,9 @@ When the password of user is updated from {project_name} and sent to LDAP, it is
 updating the password to built-in {project_name} database, when the hashing and salting is applied to the password before it is sent to DB.
 In the case of LDAP, the {project_name} relies on the LDAP server to provide hashing and salting of passwords.
 
-Most of LDAP servers (Microsoft Active Directory, RHDS, FreeIPA) provide this by default. Some others (OpenLDAP, ApacheDS) may store the passwords
-in plain-text by default and you may need to explicitly enable password hashing for them. See the documentation of your LDAP server more details.
+Most LDAP servers (Microsoft Active Directory, RHDS, FreeIPA) provide this by default. Some others (OpenLDAP, ApacheDS) may store the passwords
```
Comment (andymunro) [inline server_admin/topics/user-federation/ldap.adoc]: change the parenthetical comments to "such as...
For example, "Most LDAP servers, such as Microsoft Active Directory, RHDS, and FreeIPA, provide...

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks. One minor change.

File: server_admin/topics/user-federation/ldap.adoc (hunk 1 above)
Comment (cachescrubber) [reply server_admin/topics/user-federation/ldap.adoc]: Done.

**Review by mposolda — APPROVED** [review]
@andymunro @cachescrubber Thanks to both of you for the review and update of this PR!

**Pull request merged**